src/tests/functional/shared_test_classes/src/base/snippets_test_utils.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "pugixml.hpp"
+#include "shared_test_classes/base/snippets_test_utils.hpp"
+#include "exec_graph_info.hpp"
+
+namespace ov {
+namespace test {
+void SnippetsTestsCommon::validateNumSubgraphs() {
+    const auto& compiled_model = compiledModel.get_runtime_model();
+    size_t num_subgraphs = 0;
+    size_t num_nodes = 0;
+    for (const auto &op : compiled_model->get_ops()) {
+        auto layer_type = op->get_rt_info().at(ExecGraphInfoSerialization::LAYER_TYPE).as<std::string>();
```

Comment:
we can use rtti instead of this construction

Reply:
I can do it, no problem, but then we'll have to link sharedTestClasses against inference_engine_snippets, where the op::Subgraph is defined. Originally, I tried to avoid this dependence and used rt_info instead.

Reply:
Decided to leave it as is to avoid additional linking dependencies.